### PR TITLE
R20.1 254

### PR DESCRIPTION
**Pull request by dougkenik**
Fix for weird text rendering issues in Login Screen when using QTQuick.Controls 1

### CODE REVIEW

File: SmartSlicePlugin/stage/ui/SmartSliceLogin.qml [hunk 1]
```diff
@@ -37,9 +37,16 @@ MouseArea {
...
-        width: 0.2 * smartSliceMain.width
+        property int minimumWidth: 350
+        property int computedWidth: 0.2 * smartSliceMain.width
+
+        width: computedWidth >= minimumWidth ? computedWidth : minimumWidth
         height: childrenRect.height
 
+        Component.onCompleted: {
+            console.log("WIDTH: ", width)
+        }
```
Comment (colman-hartinger) [inline SmartSlicePlugin/stage/ui/SmartSliceLogin.qml:48]: is this meant to be here still?

**Review by colman-hartinger — CHANGES_REQUESTED** [review]
code looks good and login screen still works as expected.

File: SmartSlicePlugin/stage/ui/SmartSliceLogin.qml [hunk 2]
```diff
@@ -186,35 +178,20 @@ MouseArea {
...
-                            background: Rectangle {
-                                implicitHeight: control.height;
-                                implicitWidth: control.width;
+                        background: Rectangle {
+                            anchors.fill: parent
 
-                                border.width: UM.Theme.getSize("default_lining").width;
-                                border.color: control.hovered ? UM.Theme.getColor("setting_control_border_highlight") : UM.Theme.getColor("setting_control_border");
-                                radius: UM.Theme.getSize("setting_control_radius").width
+                            border.width: UM.Theme.getSize("default_lining").width;
```
Comment (colman-hartinger) [inline SmartSlicePlugin/stage/ui/SmartSliceLogin.qml:180]: random semicolon

File: SmartSlicePlugin/stage/ui/SmartSliceLogin.qml [hunk 3]
```diff
@@ -139,35 +142,20 @@ MouseArea {
...
-                                implicitWidth: control.width;
-
-                                border.width: UM.Theme.getSize("default_lining").width;
-                                border.color: control.hovered ? UM.Theme.getColor("setting_control_border_highlight") : UM.Theme.getColor("setting_control_border");
-                                radius: UM.Theme.getSize("setting_control_radius").width
+                        background: Rectangle {
+                            anchors.fill: parent
 
-                                color: UM.Theme.getColor("setting_validation_ok");
+                            border.width: UM.Theme.getSize("default_lining").width;
```
Comment (colman-hartinger) [inline SmartSlicePlugin/stage/ui/SmartSliceLogin.qml:148]: random semicolon

**Review by bradymadams — APPROVED** [review]

**Pull request merged**